src/main/resources/swagger/api-docs.yaml

```diff
@@ -1543,13 +1671,47 @@ paths:
         500:
           description: Internal Error
 
+  /cookie-authed/download/b/{bucket}/o/{object}:
+    get:
+      tags:
+      - Storage
+      operationId: getStorageDownload
+      summary: |
+        Download GCS object using a cookie token
+      description: |
+        **Important**: this file download does not work within the swagger UI. It is here for documentation
+        purposes. You can fill in the fields and submit; once you do so, you will get an error, but swagger
+        will populate the "Request URL" field. You can then copy and paste that field into a new tab to
+        achieve your download.
+      parameters:
+        - name: bucket
+          description: Name of the bucket in which the object resides.
+          required: true
+          type: string
+          in: path
+        - name: object
+          description: Name of the object. (be sure to urlencode)
+          required: true
+          type: string
+          in: path
+      responses:
+        200:
+          description: Success
+          schema:
+            type: file
+        404:
+          description: Not Found
+        500:
+          description: Internal Error
+      security: []
```

Comment:
The swagger UI doesn't show you the Authorization Bearer in the curl command. I am assuming this is the reason. I added googleoauth back in and now that does show up in the curl command.